bitcoin/src/blockdata/script.rs

```diff
@@ -1056,31 +1714,47 @@ impl Builder {
     /// because `OP_VERIFY` consumes an item from the stack so ignoring them would change the
     /// semantics.
     pub fn push_verify(mut self) -> Builder {
-        match self.1 {
-            Some(OP_EQUAL) => {
-                self.0.pop();
-                self.push_opcode(OP_EQUALVERIFY)
+        // "duplicated code" because we need to update `1` field
+        match opcode_to_verify(self.1) {
+            Some(opcode) => {
+                (self.0).0.pop();
+                self.push_opcode(opcode)
             },
-            Some(OP_NUMEQUAL) => {
-                self.0.pop();
-                self.push_opcode(OP_NUMEQUALVERIFY)
-            },
-            Some(OP_CHECKSIG) => {
-                self.0.pop();
-                self.push_opcode(OP_CHECKSIGVERIFY)
-            },
-            Some(OP_CHECKMULTISIG) => {
-                self.0.pop();
-                self.push_opcode(OP_CHECKMULTISIGVERIFY)
-            },
-            _ => self.push_opcode(OP_VERIFY),
+            None => self.push_opcode(OP_VERIFY),
         }
     }
 
-    /// Converts the `Builder` into an unmodifiable `Script`.
-    pub fn into_script(self) -> Script {
-        Script(self.0.into_boxed_slice())
+    /// Converts the `Builder` into `ScriptBuf`.
+    pub fn into_script(self) -> ScriptBuf {
+        self.0
+    }
+
+    /// Converts the `Builder` into script bytes
+    pub fn into_bytes(self) -> Vec<u8> {
+        self.0.into()
     }
+
+    /// Returns the internal script
+    pub fn as_script(&self) -> &Script {
+        &self.0
+    }
+
+    /// Returns script bytes 
```

Comment:
TIL: Our pre-commit hook and clippy CI job allows one character of trailing whitespace - does that mean one character of trailing whitespace is ok, clippy is wrong, or we have the clippy incantation wrong?

`cargo clippy --all-features --all-targets -- -D warnings`

Reply:
Oh weird, I though I removed all of them.